packages/terra-search-field/src/SearchField.jsx

```diff
@@ -132,17 +156,24 @@ class SearchField extends React.Component {
       customProps.className,
     ]);
 
+    const additionalInputAttributes = {};
+    if (value !== undefined) {
+      additionalInputAttributes.value = value;
+    } else {
+      additionalInputAttributes.defaultValue = defaultValue;
+    }
+
     return (
       <div {...customProps} className={searchFieldClassNames}>
         <Input
           className={cx('input')}
           type="search"
           placeholder={placeholder}
-          value={this.state.searchText}
           onChange={this.handleTextChange}
           disabled={isDisabled}
           aria-disabled={isDisabled}
           onKeyDown={this.handleKeyDown}
+          {...additionalInputAttributes}
```

Comment:
Spread `additionalInputAttributes` first so consumers can't override your props.

Reply:
additionalInputAttributes can only have props of defaultValue and value; therefore, it is ok to have them here as the user cannot override them